### DISCUSSION TITLE
Close, Minimize, & Maximize buttons

### DISCUSSION THREAD

**Discussion by Th3Whit3Wolf** [rafaelmardojai/firefox-gnome-theme#260] [Q&A]
Hi I noticed that in some of the pictures there are the system icons for closing, minimizing, and maximizing (without using a titlebar). How do I enable that (I am not using the default Adwaita theme, am using system icons).

**Comment by rafaelmardojai** [comment]
Title buttons are based on your system. So you should enable them on GNOME Tweaks.

**Comment by Th3Whit3Wolf** [comment]
I am using KDE with Mojave gtk theme.

**Comment by rafaelmardojai** [comment]
I think you can change that via terminal. 

Maybe this works for you:

```sh
gsettings set org.gnome.desktop.wm.preferences button-layout ':maximize,minimize,close'
```

**Comment by Th3Whit3Wolf** [comment]
It is currently set to `gtk-decoration-layout=close,minimize,maximize:` in `~/.config/gtk-3.0/settings.ini`

**Comment by rafaelmardojai** [comment]
If vanilla Firefox (without this theme) is not showing you the three buttons either. It might be a Firefox issue.

**Comment by Th3Whit3Wolf** [comment]
Interestingly they are in Mojave-dark although they're not the right ones . When you say without this theme do you mean with Adwaita or without a userchrome? Firefox with no userchrome/usercontent shows window decorations in the titlebar.

**Comment by rafaelmardojai** [comment]
I mean without userchrome and with titlebar disabled from Firefox Customization screen.

**Comment by Th3Whit3Wolf** [comment]
Yes after uninstalling firefox-gnome-theme they showup

**Comment by rafaelmardojai** [comment]
Can you provide some screenshots to see how it looks with the different changes?

**Comment by Th3Whit3Wolf** [comment]
![image](https://user-images.githubusercontent.com/48275422/85200847-97fd2180-b2f2-11ea-89ba-d78f356f9a1d.png)

**Comment by Th3Whit3Wolf** [comment]
![image](https://user-images.githubusercontent.com/48275422/85200974-941dcf00-b2f3-11ea-925a-f6459afd1b50.png)

**Comment by Th3Whit3Wolf** [comment]
![image](https://user-images.githubusercontent.com/48275422/85200990-b3b4f780-b2f3-11ea-9002-1035ebe17208.png)

The icon disappear after a short while.

**Comment by Th3Whit3Wolf** [comment]
![image](https://user-images.githubusercontent.com/48275422/85227820-80e42f80-b3d7-11ea-9df6-98e4c4b3c6bb.png)
Normal

**Comment by rafaelmardojai** [comment]
You're using system icons only for title buttons right?

**Comment by Th3Whit3Wolf** [comment]
Here's my userchrome.css

```css
/* Import theme */
@import "theme/gnome-theme.css";

/*
* Here you can enable other theme features not included by default.
* To enable a specific feature, uncomment its @import line by removing "/*" at
* the beginning of the line. To disable it, put the "/*" back.
*/

/* Hide the tab bar when only one tab is open (GNOMISH)
* You should move the new tab button somewhere else for this to work, because by
* default it is on the tab bar too. */
/*@import "theme/hide-single-tab.css"; /**/

/* Limit the URL bar's autocompletion popup's width to the URL bar's width (GNOMISH)
* This feature is included by default for Firefox 70+. */
/*@import "theme/matching-autocomplete-width.css"; /**/

/* Use square title buttons old style */
/*@import "theme/square-title-buttons.css"; /**/

/* Use normal width tabs */
/*@import "theme/normal-width-tabs.css"; /**/

/* Active tab high contrast */
/*@import "theme/active-tab-contrast.css"; /**/

/* Use system theme icons instead of Adwaita icons included by theme */
@import "theme/system-icons.css";

/* Allow drag window from headerbar buttons (GNOMISH) [BUGGED]
* It can activate button action, with unpleasant behavior. */
/*@import "theme/drag-window-headerbar-buttons.css"; /**/

/* Make all tab icons look kinda like symbolic icons */
/*@import "theme/symbolic-tab-icons.css"; /**/

/* Import a custom stylesheet
* Everything you add in your customChrome.css file (it doesn't exist by
* default) will be included here and preserved between updates, so you can move
* your configuration to that file if you don't like setting it up after every
* update. You can also apply your own custom styles in that file. */
@import "customChrome.css"; /**/
```

**Comment by Th3Whit3Wolf** [comment]
yes

**Comment by Th3Whit3Wolf** [comment]
Is there anything wrong with my userchrome? (I am not using a `customChrome.css`)
